README.md

```diff
@@ -252,7 +252,27 @@ See [`chain`][chain].
 
 <h4 name="chainRec"><code><a href="https://github.com/sanctuary-js/sanctuary-type-classes/blob/v0.0.0/index.js#L705">chainRec :: ChainRec m => (m x, (a -> c, b -> c, a) -> m c, a) -> m b</a></code></h4>
 
-<h4 name="reduce"><code><a href="https://github.com/sanctuary-js/sanctuary-type-classes/blob/v0.0.0/index.js#L710">reduce :: Foldable f => ((a, b) -> a) -> a -> f b -> a</a></code></h4>
+See [`chainRec`][chainRec].
+
+```javascript
+> chainRec(
+.   [],
```

Comment:
I think it's much correct to pass Type (`Array` in this case) instead of creating throwaway object here.
Also it would be sort of confusing to newcomers as it's not clear why are we passing empty array.